src/main/java/com/digital/home/controller/DailyTransactionController.java

```diff
@@ -0,0 +1,21 @@
+package com.digital.home.controller;
+
+import com.digital.home.model.DailyTransaction;
+import com.digital.home.service.DailyTransactionService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class DailyTransactionController {
+
+    @Autowired
+    private DailyTransactionService dailyTransactionService;
+
+    @GetMapping("/daily-transactions/{projectId}")
```

Comment:
Get additional parameter as date

Reply:
Since as given in the BRD, for daily transactions date is non editable, it is not added. It will take the system date in the back end.